doc/nrf/ug_thread_certification.rst

```diff
@@ -63,16 +63,19 @@ Complete the following steps to prepare for the certification tests:
 
 #. Build the certification image.
 
-   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
+   The :ref:`ot_cli_sample` sample is used as a base, modified with the following overlay files:
+   * :file:`../common/overlay-no-logging.conf`
+   * :file:`../common/overlay-ble.conf`
 
    .. code-block::
 
          cd ncs/nrf/samples/openthread/cli/
-         west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG=harness/overlay-cert.conf -DCONFIG_OPENTHREAD_LIBRARY_1_1=y
+         west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG="../common/overlay-no-logging.conf;../common/overlay-ble.conf" -DCONFIG_OPENTHREAD_LIBRARY_1_1=y
 
    .. note::
       The configuration option selects the precompiled OpenThread libraries.
-      The overlay file enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth® LE advertising.
+      The overlay :file:`../common/overlay-no-logging.conf` disables logging.
```

Comment:
Instead of a `no-logging.conf` overlay, should this be turned into a:
1. `prj.conf` (release ready build)
2. `prj_debug.conf` (debug build)

See more here: https://docs.zephyrproject.org/latest/guides/build/kconfig/setting.html#the-initial-configuration

Reply:
So Idea behind no logging overlay is not for release or debug purposes but rather turning off prompt, banner, and excessive logging that are interfering with Thread certification. I don't say that having two project configs Debug and Release is bad idea but this overlay wasn't intended for Release purposes.